blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,204 @@
+
+# How to create a Tableau dashboard:
+
+This blog will teach you how to create a Tableau dashboard in the following steps:
+* Step 1: Connect to data source
+* Step 2: Create a table with filters
+* Step 3: Create a line graph with a date filter
+* Step 4: Format the dashboard
+
+Step 1: Connect to data source
+- 
+Once you open Tableau, you need to connect to a data source. 
+We will use a Microsoft Excel file for this dashboard.
+
+![](https://i.imgur.com/UUjUMPf.jpg)
+
+
+After connection, raw data:
+
+![](https://i.imgur.com/5o0HVof.jpg)
+
+
+
+Step 2: Create a table with filters
+- 
+In the first sheet, we will create a COVID-19 stat table with state and county filters.
+This step is broken into three parts:
+* Create table
+* Remove abc column
+* Add filters
+
+**Create table**
+This is what you will see on the left side.
+Dimensions display Date, State, and County and Measures display numeric columns, such as Cases and Deaths.
+
+![](https://i.imgur.com/0GSeyCd.jpg)
+
+In the table, we would like to display Date, State, County, Cases, and Deaths columns. So we will drag those column names into "Rows" section.
+
+![](https://i.imgur.com/tLH334O.jpg)
+
+But the table looks weird.
+We need to set Cases, Deaths columns to "Discrete."
+
+![](https://i.imgur.com/YIseNHc.jpg)
+
+Once we change Cases and Deaths to Discrete, this is what it looks like!
+
+This is what we want, except that there is an Abc column.
+![](https://i.imgur.com/d3lQDWo.jpg)
+
+
+**Remove Abc column**
+This blog will cover how to remove the Abc column in the table.
+But, feel free to check out the video below if you would like to follow a video tutorial.
+https://www.youtube.com/watch?v=6FTZ6TnYaF0
+
+First, click any abc column and click Format.
+![](https://i.imgur.com/NZ6WGTx.jpg)
+
+Change Pane color to White.
+
+![](https://i.imgur.com/BcWDLzU.jpg)
+
+This is what it looks like after you change the color.
+![](https://i.imgur.com/ceout1e.jpg)
+
+Choose the column on the left of Abc and click Format.
+![](https://i.imgur.com/UqUSaab.jpg)
+
+
+Under the “Borders” (Square icon), set Column Divider Pane to None.
+![](https://i.imgur.com/ZVc527t.jpg)
+
+Shrink the width of the blank column! This is the final table!
+![](https://i.imgur.com/nTLe6cI.jpg)
+
+**Add filters**
+Now, let’s add State, County filter. (Choose Select All from List)
+
+![](https://i.imgur.com/OjVLHO0.jpg)
+
+Right click on the State filter and choose "Show Filter."
+![](https://i.imgur.com/xObIbNK.jpg)
+
+This filter shows all of the states. It's too lengthy!
+
+![](https://i.imgur.com/yMBKQLd.jpg)
+
+Right click the tiny dropdown in the State filter and change it to the form you like - I choose Single Value dropdown
+
+![](https://i.imgur.com/ENuRhRh.jpg)
+
+The lengthy filter list changed to a dropdown list!
+![](https://i.imgur.com/EPLT9LJ.jpg)
+
+Change title name and this is the final product!
+![](https://i.imgur.com/EHpACLh.jpg)
```

Comment:
I would add another screenshot here with the state filter being used, just to demonstrate what the last few steps provide here